packages/ui/src/components/Charts/SwarmPlot/Layers/LimitLineSvgLayer.tsx

```diff
@@ -17,10 +17,19 @@ export type TSwarmPlotMedianSvgLayer = {
 
 const OUTER_LINE_SIZE = 20;
 
-const sort = (a: ComputedDatum<SwarmRawDatum>, b: ComputedDatum<SwarmRawDatum>) => {
-    if (a.data.y < b.data.y) {
+const sortY = (a: ComputedDatum<SwarmRawDatum>, b: ComputedDatum<SwarmRawDatum>) => {
+    if (a.y < b.y) {
         return -1;
-    } else if (a.data.y > b.data.y) {
+    } else if (a.y > b.y) {
+        return 1;
+    }
+    return 0;
+};
+
+const sortX = (a: ComputedDatum<SwarmRawDatum>, b: ComputedDatum<SwarmRawDatum>) => {
```

Comment:
if perf is an issue, maybe explore this: https://en.wikipedia.org/wiki/Schwartzian_transform

Reply:
perf is pretty good at the moment. Will keep it in my favorite just to be sure